### ISSUE TITLE
Status of this repo? Roadmap?

### ISSUE THREAD

**Issue by Volodymyr12345678910** [cocos2d/cocos2d-x#20502]
Milestones past due for 6 months already:

![440CADB8-DA51-43F6-814E-486BAE9161E7](https://user-images.githubusercontent.com/45468910/80242490-1bd9bb00-866e-11ea-94d9-af02bb638b42.jpeg)

Roadmap? 

From the latest commits for the pas 3 months I don’t see any new features/improvements. 

Is this engine ready to start developing a new game without fear of abandonment?
What new features will be added soon? This is clear that it’s currently In pre-abandoned state.
So please tell me the truth.

@minggo @Mee-gu @PatriceJiang ?
@crazyhappygame @rh101 @drelaptop @walzer @zilongshanren @WenhaiLin @samuele3hu @pandamicro @yinjimmy @crazyhappygame @thanwah @halx99 @arsen-aar @kingsamurai123 @bolino @imtrobin @CocosRobot @makiolo

**Comment by Volodymyr12345678910** [comment]
Ignoring speaks for itself. I have drawn a conclusions. Thanks.

**Comment by halx99** [comment]
@Volodymyr-Klymenko Volodymyr-Klymenko
The official developers may do more things for CocosCreator,  that's why we create another fork [engine-x](https://github.com/c4games/engine-x) to do more research what we are good at, I also don't know what's the Roadmap of this repo.

**Comment by bolino** [comment]
I have the same doubt, in fact I made a topic regarding this on the forums but haven't gotten an answer yet:
https://discuss.cocos2d-x.org/t/v4-dev-status-could-we-have-a-roadmap/49973

**Comment by Volodymyr12345678910** [comment]
@halx99 I don't think that non official could be used for my company... also where is the guarantees for the future?

@slackmoehrle any news?
